src/overlays/actors/ovl_En_Firefly/z_en_firefly.h

```diff
@@ -5,9 +5,37 @@
 
 struct EnFirefly;
 
+typedef void (*EnFireflyActionFunc)(struct EnFirefly*, GlobalContext*);
+
+typedef enum {
+    /* 0 */ KEESE_FIRE_FLY,
+    /* 2 */ KEESE_NORMAL_FLY = 2,
+    /* 3 */ KEESE_NORMAL_PERCH,
+    /* 4 */ KEESE_ICE_FLY
+} KeeseMainType;
+
+#define KEESE_INVISIBLE (1 << 0xF)
+#define KEESE_GET_MAIN_TYPE(thisx) ((thisx)->params & 0x7FFF)
+
 typedef struct EnFirefly {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x230];
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ EnFireflyActionFunc actionFunc;
+    /* 0x18C */ u8 auraType;
+    /* 0x18D */ u8 currentType;
+    /* 0x18E */ u8 invisible;
```

Comment:
```suggestion
    /* 0x18E */ u8 isInvisible;
```
This seems to be used as a boolean right?